FirstClasses/multiples/src/multiples.erl

```diff
@@ -1,5 +1,39 @@
 -module(multiples).
 
 -export([thousand_sum/0]).
+-export([generate_input_collection/1]).
+-export([filter_collection/1]).
 
-thousand_sum() -> 1.
+thousand_sum() -> 
+    FilterCollection = filter_collection(generate_input_collection(1000)),
```

Comment:
I do like the approach, you showed. First, generate the collection, then filter it, and next sum the filtered list.

Reply:
thank you